src/applications/proxy-rewrite/redirects/disabilityRedirects.json

```diff
@@ -223,5 +223,65 @@
     "domain": "www.benefits.va.gov",
     "src": "/gibill/yellow_ribbon/yrp_list_2019.asp",
     "dest": "/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/#does-my-school-participate-in-"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/disabledvet.asp",
+    "dest": "/life-insurance/options-eligibility/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/sgli.asp",
+    "dest": "/life-insurance/options-eligibility/sgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/sglidisabled.asp",
+    "dest": "/life-insurance/options-eligibility/sgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/fsgli.asp",
+    "dest": "/life-insurance/options-eligibility/fsgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/tsgli.asp",
+    "dest": "/life-insurance/options-eligibility/tsgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/apply-for-VGLI.asp",
+    "dest": "/life-insurance/options-eligibility/vgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/vgli.asp",
+    "dest": "/life-insurance/options-eligibility/vgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/vgli_rates_new.asp",
+    "dest": "/life-insurance/options-eligibility/vgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/getvgli.asp",
+    "dest": "/life-insurance/options-eligibility/vgli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/s-dvi.asp",
+    "dest": "/life-insurance/options-eligibility/s-dvi/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/vmli.asp",
+    "dest": "/life-insurance/options-eligibility/vmli/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/insurance/abo.asp",
+    "dest": "/life-insurance/totally-disabled-or-terminally-ill/"
   }
```

Comment:
This looks great! 🎉 
![image](https://user-images.githubusercontent.com/12773166/71269172-ec0b5700-230b-11ea-81d9-aa67cc3acfc4.png)